src/index.html

```diff
@@ -11,6 +11,51 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <div class="search-bar search-bar--bg">
+      <form
+      action="#"
+      method="get"
+      class="search-bar__form"
+      data-qa="big"
+      >
+        <label for="search-bar__input">
+          <img
+          class="search-bar__icon search-bar__icon--bg"
+          src="images/Search.svg"
+          alt="search-icon"
+          >
+        </label>
+        <input
```

Comment:
make it full height, see clickable zone:
![image](https://github.com/mate-academy/layout_search-bar-airbnb/assets/86923089/3afb2dab-6daf-47e0-bda5-074fe7911d0a)